node/core/dispute-coordinator/src/status.rs

```diff
@@ -37,20 +37,19 @@ pub enum DisputeStatus {
 	/// The dispute is active and unconcluded.
 	#[codec(index = 0)]
 	Active,
-	/// The dispute has been concluded in favor of the candidate
-	/// since the given timestamp.
+	/// The dispute was in favor of the candidate since the given timestamp.
```

Comment:
I will not suggest editing this again, but to me the former version makes more sense. "was in favor" suggests that it no longer is, past tense - while obviously it is still the case until the presence. So I really don't get this suggested change.

Reply:
It's either `has been in favor of` or `was concluded in favor of`